packages/backend/src/apps/scheduler/__tests__/triggers/get-data-out-metadata.test.ts

```diff
@@ -0,0 +1,57 @@
+import { IExecutionStep } from '@plumber/types'
+
+import { describe, expect, it } from 'vitest'
+
+import getDataOutMetadata from '../../triggers/get-data-out-metadata'
+
+describe('Test getDataOutMetadata', () => {
+  it('pretty_date to convert to Date', async () => {
+    const testExecutionStep = {
+      dataOut: {
+        pretty_date: {},
+      },
+    } as unknown as IExecutionStep
+    const testMetadata = await getDataOutMetadata(testExecutionStep)
+    expect(testMetadata.pretty_date.label).toEqual('Date')
+  })
+
+  it('pretty_date to convert to Date', async () => {
```

Comment:
nit: the test name should be pretty_time instead? similar issue for the remaining ones too

Reply:
thanks for the catch!